packages/sit-onyx/src/components/OnyxRadioGroup/types.ts

```diff
@@ -22,11 +24,6 @@ export type OnyxRadioGroupProps<TValue extends SelectOptionValue = SelectOptionV
        * The selected radio button option.
        */
       modelValue?: TValue;
```

Comment:
the previous comment made a good point, could you maybe extend the comment on `RequiredMarkerProp`, like `No marker will be visible if the label of the element is hidden.`

context:
> ```
>  /**
>   * Required mode: `optional` will show an `(optional)` text after the label for optional inputs.
>   * `required` will show an `*` indicator for required inputs after the label instead.
>   * No marker will be visible if the label of the element is hidden.
>   * @default undefined By default the parents setting is used, if none is defined on any `required` is the default.
>   */
>  requiredMarker?: RequiredMarkerType;
> ```